libmamba/src/solver/libsolv/database.cpp

```diff
@@ -15,6 +15,7 @@
 #include <solv/solver.h>
 #include <spdlog/spdlog.h>
 
+#include "mamba/core/context.hpp"
```

Comment:
Yeah well that's what we want to avoid, except `core` and `api`, all the other files are context free...